frontend/ui/widget/inputdialog.lua

```diff
@@ -723,6 +724,52 @@ function InputDialog:_addScrollButtons(nav_bar)
                 end,
             })
         end
+        -- Add a button to go to the line by its number in the file
+        if self.fullscreen then
+            table.insert(row, {
+                text = "Go",
+                callback = function()
+                    local cur_line_num, last_line_num = self._input_widget:getLineNums()
+                    input_dialog = InputDialog:new{
```

Comment:
```lua
local input_dialog
input_dialog = InputDialog:new{
```
Keep it local, and this way as you reference input_dialog inside the inner functions.

Reply:
Fixed